src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -176,7 +176,7 @@ protected BoundExpression CreateUserDefinedConversion(SyntaxNode syntax, BoundEx
                 destination: conversion.BestUserDefinedConversionAnalysis.FromType,
                 diagnostics: diagnostics);
 
-            TypeSymbol conversionParameterType = conversion.BestUserDefinedConversionAnalysis.Operator.ParameterTypes[0];
+            TypeSymbol conversionParameterType = conversion.BestUserDefinedConversionAnalysis.Operator.ParameterTypes[0].TypeSymbol;
```

Comment:
>ParameterTypes [](start = 103, length = 14)

Not unique to this PR, but the terminology is somewhat confusing (I read this as using the type of the type).